frontend/src/components/GddSend/TransactionForm.vue

```diff
@@ -160,15 +160,18 @@ export default {
   },
   props: {
     balance: { type: Number, default: 0 },
+    email: { type: String, default: '' },
```

Comment:
I did this to restore the data, when the user cancels the confirmation to correct some input fields